gateway/Dockerfile

```diff
@@ -21,6 +21,9 @@ WORKDIR ${LAMBDA_TASK_ROOT}
 COPY --from=build /app/node_modules ./node_modules
 COPY --from=build /app/dist ./
 
+# Copy some static files
+COPY --from=build /app/internal-services/logout/schema.graphql ./internal-services/logout/schema.graphql
```

Comment:
we only copy the typescript build above (/app/dist), files that are not ts are ignored so we need to manually copy them